DEVELOPMENT.md

```diff
@@ -21,7 +23,7 @@ This is the simplest configuration for developers to start with.
 Occasionally, new package dependencies or schema changes will necessitate
 maintenance. To non-destructively update your development stack at any time:
 1. Run `docker compose pull`
-2. Run `docker compose build --pull --no-cache`
+2. Run `docker compose build --pull --no-cache --build-arg USERID=$(id -u) --build-arg GROUPID=$(id -g) --build-arg LOGIN=$(id -n -u)` (omitting the `--build-arg` arguments if you did so in Step 1 of *Initial Setup* above).
```

Comment:
This didnt work for me 

<details><summary>Fail copypaste</summary>

```
austin@fancy ~/devel/dandi-archive (branch detox) docker compose build --pull --no-cache --build-arg USERID=$(id -u) --build-arg GROUPID=$(id -g) --build-arg LOGIN=$(id -n -u)
WARN[0000] /home/austin/devel/dandi-archive/docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion 
WARN[0000] /home/austin/devel/dandi-archive/docker-compose.override.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion 
[+] Building 25.8s (10/33)                            docker:default
 => [django internal] load build definition from django.Docker  0.0s
 => => transferring dockerfile: 2.62kB                          0.0s
 => [celery internal] load build definition from django.Docker  0.0s
 => => transferring dockerfile: 2.62kB                          0.0s
 => [celery internal] load metadata for docker.io/library/pyth  0.6s
 => [celery internal] load .dockerignore                        0.0s
 => => transferring context: 2B                                 0.0s
 => [django internal] load .dockerignore                        0.0s
 => => transferring context: 2B                                 0.0s
 => CACHED [celery  1/16] FROM docker.io/library/python:3.11-s  0.0s
 => [django internal] load build context                        0.0s
 => => transferring context: 5.23kB                             0.0s
 => [django  2/16] RUN apt-get update &&     apt-get install   24.7s
 => [celery internal] load build context                        0.0s
 => => transferring context: 5.23kB                             0.0s
 => ERROR [celery  3/16] RUN getent group 1000 || addgroup --g  0.4s
------                                                               
 > [celery  3/16] RUN getent group 1000 || addgroup --gid 1000 root: 
0.289 addgroup: The group `root' already exists.                     
------                                                               
failed to solve: process "/bin/sh -c getent group ${GROUPID} || addgroup --gid ${GROUPID} ${GROUPNAME}" did not complete successfully: exit code: 1
```
</details>

Pretty sure we just need to add GROUPNAME, since it was trying to use `root`. I tried the suggestion below, and the build progressed past this point.

```suggestion
2. Run `docker compose build --pull --no-cache --build-arg USERID=$(id -u) --build-arg GROUPID=$(id -g) --build-arg LOGIN=$(id -n -u) --build-arg GROUPNAME=$(id -gn)` (omitting the `--build-arg` arguments if you did so in Step 1 of *Initial Setup* above).
```

Reply:
There is 2ada467 in the commit stream. Can you pull this branch and try again?

Reply:
@waxlamp yeah this line needs to match the line you changed in that commit

Reply:
Ah got it, see 026dc13.